webpack/components/extensions/HostDetails/Cards/HostCollectionsCard/HostCollectionsCard.scss

```diff
@@ -0,0 +1,14 @@
+.host-collection-card-body {
+  margin: -50px;
+
+  .pf-c-empty-state__icon {
+    font-size: 2em;
+  }
+  .pf-m-lg {
+    margin: -20px;
+  }
+  .pf-m-secondary {
+    margin-top: 5px;
+    font-size: 0.75em;
+  }
+}
```

Comment:
```suggestion
}

.pf-c-card__header {
  z-index: 99;
}
```

Reply:
Seems this `z-index` makes no difference here.

Reply:
The `z-index` is needed because of the `position: relative`. Once you add that you'll see that you can't click on the kebab without adding the z-index.